### PR TITLE
RNs-4.10.44 Added with bug fix update

### PR DESCRIPTION
**Pull request by Wiharris**
**RNs-4.10.44** 
Added with bug fix update

**Version(s):**
PR only applies to enterprise-4.10

**Issue:**
https://issues.redhat.com/browse/OSDOCS-4628

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/RNs-4.10.44/release_notes/ocp-4-10-release-notes.html#ocp-4-10-44

**Additional information:**
Errata links will not show as advisories have not yet `shipped live`

### CODE REVIEW

**Comment by Wiharris** [comment]
/remove-label peer-review-needed

Removing label to add backport.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -3658,4 +3658,33 @@ $ oc adm release info 4.10.43 --pullspecs
...
+
+[source,terminal]
+----
+$ oc adm release info 4.10.44 --pullspecs
+----
+
+[id="ocp-4-10-44-enhancements"]
+==== Enhancements
+
+* IPv6 unsolicited neighbor advertisements and IPv4 gratuitous address resolution protocol now default on the SR-IOV CNI plug-in. Pods created with the Single Root I/O Virtualization (SR-IOV) CNI plug-in, where the IP address management CNI plug-in has assigned IPs, now send IPv6 unsolicited neighbor advertisements and/or IPv4 gratuitous address resolution protocol by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh ARP/NDP caches with the correct information. For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
```
[the remark this answers is not in the document]
Comment (tmalove) [reply release_notes/ocp-4-10-release-notes.adoc:3680]: Unless 'ARP/NDP' is commonly used, I suggest changing this to 'ARP or NDP'.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -3658,4 +3658,33 @@ $ oc adm release info 4.10.43 --pullspecs
...
+
+[id="ocp-4-10-44-bug-fixes"]
+==== Bug fixes
+
+* Previously, object storage instances responded with `204 No Content` when there was no content to display. The {rh-openstack-first} SDK used in {product-title} did not handle 204s correctly. With this update, the installation program works around the issue when there are zero items to list. (link:https://issues.redhat.com/browse/OCPBUGS-4160[OCPBUGS-4160])
+
+[id="ocp-4-10-44-upgrading"]
+==== Updating
+
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
```
Comment (tmalove) [inline release_notes/ocp-4-10-release-notes.adoc:3690]:
```suggestion
To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
```

Comment (tmalove) [reply release_notes/ocp-4-10-release-notes.adoc:3690]: I did not see a section with the title 'Updating a cluster within a minor version by using the CLI'. I might have missed it though!

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (Wiharris) [reply release_notes/ocp-4-10-release-notes.adoc]: Hi @tmalove,

Thanks for your review and I agree! I have made the above updates! :)

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -3658,4 +3658,33 @@ $ oc adm release info 4.10.43 --pullspecs
...
+
+[id="ocp-4-10-44-enhancements"]
+==== Enhancements
+
+* IPv6 unsolicited neighbor advertisements and IPv4 gratuitous address resolution protocol now default on the SR-IOV CNI plug-in. Pods created with the Single Root I/O Virtualization (SR-IOV) CNI plug-in, where the IP address management CNI plug-in has assigned IPs, now send IPv6 unsolicited neighbor advertisements and/or IPv4 gratuitous address resolution protocol by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh ARP/NDP caches with the correct information. For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
+
+[id="ocp-4-10-44-bug-fixes"]
+==== Bug fixes
+
+* Previously, object storage instances responded with `204 No Content` when there was no content to display. The {rh-openstack-first} SDK used in {product-title} did not handle 204s correctly. With this update, the installation program works around the issue when there are zero items to list. (link:https://issues.redhat.com/browse/OCPBUGS-4160[OCPBUGS-4160])
```
[the remark this answers is not in the document]
Comment (Wiharris) [reply release_notes/ocp-4-10-release-notes.adoc]: Done! :)

**Pull request closed without merging**